tutorials/scripting/gdextension/gdextension_docs_system.rst

```diff
@@ -116,9 +116,9 @@ Currently they supported tags for the GDExtension documentation system are:
     - ``[s]{text}[/s]``
 
   * - | **kbd**
-      | Makes ``{text}`` use the mono font and styles the text color and background like a shortcut.
+      | Indicates ``{text}`` is a keyboard shortcut.
```

Comment:
Is it possible to keep "Makes..." for consistency?

Reply:
"Makes text use a grey beveled background, indicating a keyboard shortcut."?

Reply:
I went ahead with that, feel free to resolve if that sounds better to you.